lib/engine/game/g_1854/step/merge_minors.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1854
+      module Step
+        class MergeMinors < Engine::Step::Base
+          def actions(entity)
+            return [] unless entity == current_entity
+            return [] unless entity.minor?
+            return [] unless @game.minor_mergers_allowed?
+
+            actions << ['merge']
+            # TODO: pass is not working, keeps same minor as current entity
```

Comment:
What's the problem here? Nothing changes when you attempt to pass, you're stuck on this step?
This _might_ be something to do with your definition of `active?` below. Try adding a check against `@passed` in that method.

Reply:
Yeah, I was ready to ask to defer this until a future PR since I had never seen it working, but you're right it is the definition of `active?`  I'll add the @ passed  check in there.

Reply:
Testing shows pass is working and pass button not present when selecting A/B/C